mineral-ui/mineral-ui
# Update Dependencies

### Description

* Update dependencies
  * No packages with breaking changes
  * Excludes `React` and `Enzyme` as those will be separate efforts
* Update test snapshots

### Motivation and context

Updated dependencies are better than outdated dependencies.

### How to test

1. `npm test` - no failing tests
1. `npm start` - starts and hot reload works
1. `npm run build` - builds as expected
1. Build (faux release) and check it out in a sandbox app
1. Review the snapshot changes - looks like an indentation change
1. Review the website with the dev console open and see if there is anything amiss
1. Other?

### Types of changes

- Other (provide details below)

Updated dependencies and updated test snapshots

### Checklist
